index.html

```diff
@@ -16,37 +16,101 @@
+                    <a href="#">About</a>
+                </li>
+                <li class="actual-links">
+                    <a href="#">Contact</a>
+                </li>
+            </ul>
         </nav>
         <section class="container">
-            <h1 class="name">Charlie Giorgania</h1>
-            <p class="description">
-                Hello! I am a software developer! I can
-                help you build a product, feature or
-                website. Taka a look of my works.
-                If you like what you see and have a
-                project you need coded, don’t hesitate
-                and contact me.</p>
-            <button type="button" class="actual-button">
-                Start collaboration
-            </button>
-            <div class="bottom-image"></div>
+            <div class="name-container">
+                <h1 class="name">Charlie Giorgania</h1>
+                <p class="description">
+                    Hello! I am a software developer! I can
+                    help you build a product, feature or
+                    website. Taka a look of my works.
+                    If you like what you see and have a
+                    project you need coded, don’t hesitate
+                    and contact me.</p>
+                <button type="button" class="actual-button">
+                    Start collaboration
+                </button>
+            </div>
+            <div class="side-image"></div>
         </section>
+        <div class="bottom-image"></div>
         <section class="projects-section">
-            <h2 class="projects-title">Projects</h2>
-            <div class="projects-template">
-                <h3 class="project-name">Project name goes here</h3>
-                <div class="project-image">
-                    <img class="actual-image" src="images/image1.png"
-                        alt="This is an image of someone typing on a computer">
+            <div class="first-grid">
+                <h2 class="projects-title">Projects</h2>
+                <div class="projects-template-1">
+                    <div class="project-image">
+                        <img class="actual-image" src="images/image1.png"
+                            alt="This is an image of someone typing on a computer">
+                    </div>
+                    <div class="project-container">
+                        <h3 class="project-name">Project name goes here</h3>
+                        <ul class="project-tags">
+                            <li class="languages">HTML/CSS</li>
+                            <li class="languages">Ruby on Rails</li>
+                            <li class="languages">JavaScript</li>
+                        </ul>
+                        <button type="button" class="project-button">
+                            See this project <span class="arrow-icon"></span>
+                        </button>
+                    </div>
```

Comment:
- [ ] Your hover element styling is not quite similar to the Figma design, kindly adjust your styling according to the Figma design.
- Figma design:
![image](https://user-images.githubusercontent.com/33767951/139514206-b108ac7b-4301-40b1-983d-d60724eb83bf.png)
- Your styling:
![image](https://user-images.githubusercontent.com/33767951/139514238-a7db8886-b56d-4ac8-bf16-06c2a7c60565.png)